diff --git a/pod/main/static/js/main.js b/pod/main/static/js/main.js
index d05f62fa..827c6e11 100644
--- a/pod/main/static/js/main.js
+++ b/pod/main/static/js/main.js
@@ -400,11 +400,15 @@ if (ownerboxnavbar) {
   ownerboxnavbar.addEventListener("keyup", function () {
     if (ownerboxnavbar.value && ownerboxnavbar.value.length > 2) {
       var searchTerm = ownerboxnavbar.value;
-      url = "/ajax_calls/search_user?term=" + searchTerm;
+      let data = new FormData();
+      data.append("term", searchTerm);
+      data.append("csrfmiddlewaretoken", Cookies.get("csrftoken"));
+      url = "/ajax_calls/search_user/";
       fetch(url, {
-        method: "GET",
+        method: "POST",
+        body: data,
         headers: {
-          
+          Accept: "application/json",
           "X-Requested-With": "XMLHttpRequest"
         },
       })
@@ -413,7 +417,7 @@ if (ownerboxnavbar) {
           let accordion = document.getElementById("accordion");
           accordion.innerHTML = "";
           data.forEach((elt) => {
-            accordion.append(
+            accordion.innerHTML +=
               '<li><a href="' +
                 urlvideos +
                 "?owner=" +
@@ -424,8 +428,7 @@ if (ownerboxnavbar) {
                 elt.last_name +
                 (!HIDE_USERNAME
                   ? " (" + elt.username + ")</a></li>"
-                  : "</a></li>")
-            );
+                  : "</a></li>");
           });
         });
     } else {
